handlers_test.go

```diff
@@ -0,0 +1,152 @@
+package main
+
+import (
+	"fmt"
+	"github.com/application-research/estuary/node"
+	"github.com/application-research/estuary/stagingbs"
+	"github.com/application-research/estuary/util/gateway"
+	"github.com/application-research/filclient"
+	"github.com/filecoin-project/lotus/api"
+	"github.com/labstack/echo/v4"
+	"github.com/stretchr/testify/assert"
+	"github.com/whyrusleeping/memo"
+	"go.opentelemetry.io/otel/trace"
+	"gorm.io/gorm"
+	"os"
+	"testing"
+)
+
+func skipCI(t *testing.T) {
+	if os.Getenv("CI") != "" {
+		t.Skip("Skipping testing in CI environment")
+	}
+}
+
+func TestServer_handleAdd(t *testing.T) {
+	skipCI(t)
+	type fields struct {
+		tracer      trace.Tracer
+		Node        *node.Node
+		DB          *gorm.DB
+		FilClient   *filclient.FilClient
+		Api         api.Gateway
+		CM          *ContentManager
+		StagingMgr  *stagingbs.StagingBSMgr
+		gwayHandler *gateway.GatewayHandler
+		cacher      *memo.Cacher
+	}
+	type args struct {
+		c echo.Context
+		u *User
+	}
+	tests := []struct {
+		name    string
+		fields  fields
+		args    args
+		wantErr assert.ErrorAssertionFunc
+	}{
+		// TODO: Add test cases.
```

Comment:
I don't think this style of tests will work in this case, because for every test case we would have to set the DB and other things all over again. I'm working on some tests that mock the database, but the roadmap for those are basically: autoretrieve > refactor > tests. So we might want to remove this file for now.